public/components/report_definitions/create/create_report_definition.tsx

```diff
@@ -77,21 +79,26 @@ interface Cron {
   };
 }
 
-export interface reportDefinitionParams {
-  report_params: reportParamsType;
-  delivery: deliveryType;
-  trigger: triggerType;
+export interface ReportDefinitionParams {
+  report_params: ReportParamsType;
+  delivery: DeliveryType;
+  trigger: TriggerType;
 }
 
-export interface timeRangeParams {
+export interface TimeRangeParams {
   timeFrom: Date;
   timeTo: Date;
 }
 
-export function CreateReport(props: { [x: string]: any; setBreadcrumbs?: any; httpClient?: any; chrome: any }) {
+export function CreateReport(props: {
```

Comment:
nit, i know this naming is not from this PR. but use `createReport`? And is it possible to start adding types for these any?